### DISCUSSION TITLE
Where is ModChart Lua File?

### DISCUSSION THREAD

**Discussion by Fefe543** [KadeArchive/Kade-Engine#230] [Q&A] [answered]
before anything:
yes i'm dumb
So, basically, where is the ModChar Lua File? I go to assets, there's literally no data folder
![image](https://user-images.githubusercontent.com/77512259/115071211-574b4600-9ecc-11eb-9c64-33368a48556c.png)
help

**Comment by isakube** [comment, marked as answer]
ayyyyy

preload/data/(songname) and the modchart should go there

**Reply by Fefe543** [reply]
oml sorry i was just fucking dumb lollll
